macrosynergy/download/jpmaqs.py

```diff
@@ -0,0 +1,536 @@
+""" JPMaQS Download Interface """
+
+from typing import List, Optional, Tuple
+import pandas as pd
+import numpy as np
+from collections import defaultdict
+import warnings
+from macrosynergy.download import dataquery
+import logging
+import sys
+
+# check if parent logger exists. else, create it.
+if not logging.getLogger().hasHandlers():
+    logging.basicConfig(
+        stream=sys.stderr,
+        level=logging.WARNING,
+        format="%(asctime)s - %(levelname)s - %(funcName)s - %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S %p",
+    )
```

Comment:
Never modify loggers inside code :)

Reply:
Even if it means users won't see the output.

Reply:
Removed